### PR TITLE
updates to the Docker readme files

### PR DESCRIPTION
**Pull request by rfiorella**
Updates docker readme files:
1) removes all mentions of Travis CI since it is no longer being used
2) adds information about GitHub actions
3) adds information about building containers for multiple architectures.

### CODE REVIEW

**Comment by rfiorella** [comment]
@jd-moulton did you want to review this? Or okay to merge?

**Review by jd-moulton — APPROVED** [review]
These updates look good, definitey an improvement.  At this stage they are still scripts that need editing for new deployments (editing of some variables), and I wonder if its worth adding argument parsing at somepoint so the scripts themselves don't really change.

File: Docker/README-Docker.md [hunk 1]
```diff
@@ -1,44 +1,52 @@
...
+If you are not pushing your image to the metsi/amanzi-tpls DockerHub repository, this is as far as you need to go for the TPLs image.
 
-```docker tag metsi/amanzi-tpls:latest metsi/amanzi-tpls:<insert version number here>```
- 
-```docker push metsi/amanzi-tpls```
+```
+docker tag metsi/amanzi-tpls:latest metsi/amanzi-tpls:<insert version number here>
+docker push metsi/amanzi-tpls
+```
    - For this step, you will need access to the DockerHub repository. Login information can be obtained from David Moulton or Alexis Perry.
```
Comment (jd-moulton) [inline Docker/README-Docker.md:40]: Alexis Perry got this started but she left the team.  Please remove her as a contact.

File: Docker/README-Docker.md [hunk 2]
```diff
@@ -1,44 +1,52 @@
 
-## How to use the Docker files contained in this directory
+# How to use the Docker files contained in this directory
 
---------------------------------------------------------
-1. How to build and update the TPL Docker image
---------------------------------------------------------
+Currently, the Amanzi/ATS Docker image stack consists of three different images:
```
Comment (jd-moulton) [inline Docker/README-Docker.md:4]: After a lot of debate we settled on Amanzi-ATS over Amanzi/ATS.

**Comment by rfiorella** [comment]
Sounds good, I'll make the requested changes and then merge

**Comment by rfiorella** [comment]
Argument parsing added to scripts, ready for a re-review if you wanted to take a look.

**Comment by rfiorella** [comment]
@jd-moulton Okay to merge? Or would you like to see additional changes here?

**Review by jd-moulton — APPROVED** [review]
I think the changes look good and these are ready to go.  Although, I think we'll keep iterating in the future as we run the risk of making these overly complicated to make them flexible enough to cover everyone's needs.  So an open question remains of where to draw the line to balance simplicity/readability and flexibility.

**Comment by jd-moulton** [comment]
If you can do this Monday morning, it would be great to also cherry-pick it over to the amanzi-1.5 branch as we are likely to make a 1.5.1 release Monday or Tuesday.

**Comment by rfiorella** [comment]
These 11 commits have also been cherry-picked to the amanzi-1.5 branch. 
With that, I will merge and close this PR.

**Pull request merged**